apps/cf-ai-backend/src/index.ts

```diff
@@ -511,6 +511,49 @@ app.post(
 
     const userMessage: CoreMessage = { role: "user", content: prompt };
 
+    // Check for any suspicious patterns to avoid system prompt leak
+    const suspiciousPatterns = [
```

Comment:
update the patterns as you see fit

Reply:
there's an infinite set of patterns to track here. it's impossible to get all of them. To be very honest, this simply does not make sense.